stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -466,9 +466,9 @@ ::util::Status BFChassisManager::VerifyChassisConfig(
         << " has the reserved CPU port ID (" << kCpuPortId << ").";
     CHECK_RETURN_IF_FALSE(singleton_port.slot() > 0)
         << "No valid slot in " << singleton_port.ShortDebugString() << ".";
-    CHECK_RETURN_IF_FALSE(singleton_port.port() > 0)
+    CHECK_RETURN_IF_FALSE(singleton_port.port() >= 0)
         << "No valid port in " << singleton_port.ShortDebugString() << ".";
-    CHECK_RETURN_IF_FALSE(singleton_port.channel() == 0)
+    CHECK_RETURN_IF_FALSE(singleton_port.channel() >= 0)
```

Comment:
We support channelization of port for Tofino

Reply:
this is also wrong for tofino model... reverting
```suggestion
    CHECK_RETURN_IF_FALSE(singleton_port.channel() == 0)
```